backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -56,33 +58,48 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Request parameter values
-    // TODO: Check input values are filled (before allowing to submit)
-    UserService userService = UserServiceFactory.getUserService();
-    String email = userService.getCurrentUser().getEmail();
-    if (email == null) {
-      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid email.");
-      return;
+
+    // Retrieve body of HTTP request
+    StringBuffer requestBuffer = new StringBuffer();
```

Comment:
Prefer to use StringBuilder instead of StringBuiffer except when the object needs to be synchronized across threads.